src/analyzing/Constant_propagation.ml

```diff
@@ -641,3 +641,4 @@ let propagate_dataflow lang ast =
       (* Top-level function. No need to use CFG_build.cfg_of_fdef here. *)
       let flow = CFG_build.cfg_of_stmts xs in
       propagate_dataflow_one_function lang [] flow
+[@@trace_trace]
```

Comment:
this @trace_trace is ugly @emjin .
What about just [@@tracing] ?

Reply:
`@@trace` is the info level 😂. `@@trace_trace` indicates "trace at the trace level" (similar to `@@trace_debug`). It's weird because the official syntax calls this level `trace`, meaning "show all traces".

I would actually prefer to include the trace level as an "argument" but it took too long trying to figure out how to do that so I gave up: https://linear.app/semgrep/issue/SAF-1115/support-a-more-extensible-annotation-for-trace-levels